Autocomplete.md

```diff
@@ -0,0 +1,219 @@
+
+&nbsp;
+
+Normalized Company, Major, and School entities maintain an internal "popularity" score and results for these taxonomies will be returned in descending sorted order according to this score. Other taxonomies are not scored in this way and will return their results in alphabetical order.
+
+All responses will be returned as JSON.
+
+## Examples
+
+Following are several request-response examples that demonstrate usage of the service. These examples are accompanied with links to the CB API tester page that demonstrate the requests in action. Note that you must be connected to the internal CB network and add the "API Management" Okta application to your Okta profile in order to access the API tester page.
+
+**Carotene V3 autocompletion for "account" with maximum 3 results [link](https://apimanagement.cbplatform.link/#routes/tester?preURL=https%3A%2F%2Fapi.careerbuilder.com%2F&postURL=core%2Fautocomplete%2Fcarotenev3%3Fquery%3Daccount%26limit%3D3&method=get&contentType=application%2Fjson&acceptType=application%2Fjson&version=1.0&region=productionus&flow=client_credentials&userDid=&accountDid=&headers=&body=)**
+
+**Request**
+
+`https://www.api.careerbuilder.com/core/autocomplete/carotenev3?query=account&limit=3`
+
+**Response**
+```json
+{
+  "data": {
+    "results": [
+      {
+        "suggestion": "Accountant",
+        "id": "13.0"
+      },
+      {
+        "suggestion": "Account Leader",
+        "id": "41.10310"
+      },
+      {
+        "suggestion": "Account Liaison",
+        "id": "41.30981"
+      }
+    ]
+  }
+}
+```
+
+&nbsp;
+
+**Normalized Company autocompletion for "careerb" [link](https://apimanagement.cbplatform.link/#routes/tester?preURL=https%3A%2F%2Fapi.careerbuilder.com%2F&postURL=core%2Fautocomplete%2Fcompany_norm%3Fquery%3Dcareerb&method=get&contentType=application%2Fjson&acceptType=application%2Fjson&version=1.0&region=productionus&flow=client_credentials&userDid=&accountDid=&headers=&body=)**
+
+**Request**
+
+`https://www.api.careerbuilder.com/core/autocomplete/company_norm?query=careerb`
+
+**Response**
+```json
+{
+  "data": {
+    "results": [
+      {
+        "suggestion": "Careerbuilder, LLC",
+        "id": "NC842f69a0-b621-48f6-b53c-9b3301e9aab7"
+      }
+    ],
+    "data_version": "2018-03-15"
+  }
+}
+```
+
+&nbsp;
+
+**Normalized School autocompletion for "west", limited to postsecondary schools and maximum 10 results [link](https://apimanagement.cbplatform.link/#routes/tester?preURL=https%3A%2F%2Fapi.careerbuilder.com%2F&postURL=core%2Fautocomplete%2Fschool_norm%3Fquery%3Dwest%26school_level%3Dpostsecondary%26limit%3D10&method=get&contentType=application%2Fjson&acceptType=application%2Fjson&version=1.0&region=productionus&flow=client_credentials&userDid=&accountDid=&headers=&body=)**
+
+**Request**
+
+`https://www.api.careerbuilder.com/core/autocomplete/company_norm?query=west&school_level=postsecondary&limit=10`
```

Comment:
The route here still has `company_norm` instead of `school_norm` for this example. (The link to API Tester is good though)

Reply:
good catch, thanks Vito